src/components/tabs/_tabs.scss

```diff
@@ -29,7 +27,7 @@ $include-html: false !default;
       border-right: 1px solid $tabSeparatorColor;
       color: $tabTextColor;
       cursor: pointer;
-      font-family: $tabsFontFamily;
+      font-weight: bold;
```

Comment:
Nah, it can be a mixin with a font name. It does not make sense for me to move it to a variable